test/scripts/e2e_subs/goal/goal.py

```diff
@@ -112,6 +112,18 @@ def open_kmd(self, algodata, kmd_address=None):
             kmd_address = "http://" + net
         return algosdk.kmd.KMDClient(kmd_token, kmd_address)
 
+    def get_endpoint_info(self) -> dict:
```

Comment:
@jannotti - I needed this in order to test the temporary REST endpoints while pausing my debugger.

Reply:
I don't quite get it.  The info was already available as `goal.algod.algod_address` now it's reformatted so you can say `val = goal.get_endpoint_info(); val["algod"]["url"]` ?

Reply:
👍

Reply:
Just making it a bit more straightforward to get all the info, but I'll revert.